test/tests/functional/pbs_cray_suspend_resume.py

```diff
@@ -226,66 +236,56 @@ def test_preempt_STF(self):
         min_walltime can't be satisfied.
         """
         qname = 'highp'
-        queues = self.server.status(QUEUE)
-        for q in queues:
-            if 'id' in q and q['id'] == qname:
-                rc = self.server.manager(MGR_CMD_DELETE, QUEUE, None, qname)
-                self.assertEqual(rc, 0)
-                break
         a = {'queue_type': 'execution'}
         self.server.manager(MGR_CMD_CREATE, QUEUE, a, qname)
         a = {'enabled': 'True', 'started': 'True', 'priority': '150'}
         self.server.manager(MGR_CMD_SET, QUEUE, a, qname)
 
         # Reserve all the compute nodes
         vlist = []
-        try:
-            vnl = self.server.filter(
-                VNODE, {'resources_available.vntype': 'cray_compute'})
-            vlist = vnl["resources_available.vntype=cray_compute"]
-        except Exception:
-            sys.exc_clear()
-            self.logger.info("There are no cray_compute vnodes present.")
-        numvnl = len(vlist)
+        vnl = self.server.filter(
+            VNODE, {'resources_available.vntype': 'cray_compute'})
+        vlist = vnl["resources_available.vntype=cray_compute"]
+        nv = len(vlist)
+        self.assertNotEqual(nv, 0, "There are no cray_compute vnodes present.")
 
         now = time.time()
-        resv_start = now + 180
+        resv_start = now + 100
```

Comment:
Based on this new calculation, you would have to change the docstring. Since there could be slowness on running PTL on Cray, I'll suggest you change the timeout for this function to aleast 240 seconds.